### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by iam-mhaseeb**
- Added pull request template for future pull requests.

### CODE REVIEW

**Review by zubair-arbi — APPROVED** [review]
👍

**Review by etechdevai — APPROVED** [review]
👍

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+**JIRA:** 
+https://edlyio.atlassian.net/browse/EDLY-****
+
+**Testing instructions:**
+
+1. Open page A
+2. Do thing B
+3. Expect C to happen
+4. If D happend instead - check failed.
```
Comment (burhandodhy) [inline .github/pull_request_template.md]: happened

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- [ ] CI build is green
+- [ ] Documentation updated (not only docstrings)
+- [ ] Commits are (reasonably) squashed
+- [ ] Translations are updated
+
+**Post merge:**
+- [ ] Delete working branch (if not needed anymore)
+
+
+**Note:** Please add screenshots for any view able change.
```
Comment (burhandodhy) [inline .github/pull_request_template.md]: viewable

**Review by burhandodhy — CHANGES_REQUESTED** [review]
Please fix the typo

**Review by burhandodhy — APPROVED** [review]
👍

**Pull request merged**